tests/acceptance/features/bootstrap/PasswordPolicyContext.php

```diff
@@ -899,6 +899,33 @@ public function theDaysUntilLinkExpiresWithoutPasswordShouldBeSetTo(
 		);
 	}
 
+	/**
+	 * @Then /^the email address "([^"]*)" should not have received any emails$/
+	 *
+	 * @param string $emailAddress
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function shouldNotReceiveAnyEmails(string $emailAddress):void {
```

Comment:
core EmailContext.php has `public function assertThatEmailDoesntExistWithTheAddress(string $address):void`

`Then the email address :address should not have received an email`

We should be able to use that step. Please check that that step works well (and refactor it if needed).